README.md

```diff
@@ -75,5 +75,38 @@ re-run `docker-compose run app bundle install` or the `rake` commands
 listed above.  Failing these options, try rebuilding with
 `docker-compose build`.
 
+## Building native apps
+
+#### Prerequisites
+
+You have to be on OS X.
+
+You should also have Node.js and `npm` installed.
+[Here is a good set of instructions for OS X](http://shapeshed.com/setting-up-nodejs-and-npm-on-mac-osx/)
+
+You should also install the [following things](https://facebook.github.io/react-native/docs/getting-started.html):
+
+    `brew install watchman`
+    `brew install flow`
+    `npm install -g react-native cli`
```

Comment:
sorry, merged? is that the right term? accepted? i figured it's a "pull request" so someone would, you know, pull it

and yes that is a typo, it should be `react-native-cli`. sorry about the build warnings.